packages/react-next/src/components/Checkbox/__snapshots__/Checkbox.test.tsx.snap

```diff
@@ -63,10 +63,8 @@ exports[`Checkbox renders checked correctly 1`] = `
         @media screen and (-ms-high-contrast: active){.ms-Fabric--isFocusVisible &:focus + label::before {
           outline: 1px solid ActiveBorder;
         }
-    id="checkbox-1"
-    onBlur={[Function]}
+    id="checkbox-2"
```

Comment:
Checkbox.tests might need beforeEach resetIds as well. Ids like this should not be random depending on load order

Reply:
The old Checkbox would always define onFocus and onBlur.

In the function component refactor, these 2 are only set if the user passes in `inputProps`. So the change is expected.

We don't know how this passed in Michael's refactor though, so maybe a bag merge was involved.